### ISSUE TITLE
Example Grafana dashboards

### ISSUE THREAD

**Issue by a-bali** [sapslaj/morbius#10]
Hello,
I have just deployed morbius (using victoriametrics instead of prometheus and loki) and hope to get some meaningful analyses / visualizations out of it. Could you please add some example Grafana dashboards for the metrics / log entries to help getting started?

Thanks!

**Comment by sapslaj** [comment]
For sure. I've been meaning to do that for a long time but just haven't gotten around to it 😅. I'm also using VictoriaMetrics instead of Prometheus so I can at least confirm it works totally fine there.

Assuming a relatively vanilla configuration, check out `{job="neflow"}` in Loki or any of the `netflow_`* metrics in VM/Prometheus (`netflow_bytes`, `netflow_flows`, `netflow_packets`, and `netflow_ip_info` if you have that enabled). I know this isn't a replacement for good docs, but trying to help out a little bit in the meantime.

**Comment by a-bali** [comment]
Yeah I have those metrics but I find it difficult to create any meaningful aggregated visualization from them (e.g. traffic per destination ip or traffic per source countries etc.).
